bucket/firefox-nightly.json

```diff
@@ -23,5 +25,27 @@
             "firefox.exe",
             "Firefox Nightly"
         ]
-    ]
+    ],
+    "checkver": {
+        "url": "https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/",
+        "re": "firefox-([\\d\\.]+a\\d+)\\.en-US"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-$version.en-US.win64.zip",
+                "hash": {
+                    "find": "([a-f0-9]{64})\\ssha256\\s\\d+\\sfirefox-$version.en-US.win64.zip",
+                    "url": "https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-$version.en-US.win64.checksums"
+                }
+            },
+            "32bit": {
+                "url": "https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-$version.en-US.win32.zip",
+                "hash": {
+                    "find": "([a-f0-9]{64})\\ssha256\\s\\d+\\sfirefox-$version.en-US.win32.zip",
+                    "url": "https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-$version.en-US.win32.checksums"
```

Comment:
@r15ch13 What about present new substitution variable like `$urlNoExt` which would just strip file extension from URL. There are few manifests which would benefit from it.